#! /bin/sh /usr/share/dpatch/dpatch-run

## 16_vdr_1.3.45_diseqc.conf-fix.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: disable unknown entries in diseqc.conf

@DPATCH@
diff -urNad vdr-1.3.45/diseqc.conf /tmp/dpep.qpQWLe/vdr-1.3.45/diseqc.conf
--- vdr-1.3.45/diseqc.conf	2006-01-14 12:14:09.000000000 +0100
+++ /tmp/dpep.qpQWLe/vdr-1.3.45/diseqc.conf	2006-03-26 22:38:36.847934500 +0200
@@ -32,10 +32,10 @@
 S19.2E  11700 H  9750  t V W15 [E0 10 38 F2] W15 A W15 t
 S19.2E  99999 H 10600  t V W15 [E0 10 38 F3] W15 A W15 T
 
-S21.5E  11700 V  9750  t v W15 [E0 10 38 F4] W15 B W15 t
-S21.5E  99999 V 10600  t v W15 [E0 10 38 F5] W15 B W15 T
-S21.5E  11700 H  9750  t V W15 [E0 10 38 F6] W15 B W15 t
-S21.5E  99999 H 10600  t V W15 [E0 10 38 F7] W15 B W15 T
+# S21.5E  11700 V  9750  t v W15 [E0 10 38 F4] W15 B W15 t
+# S21.5E  99999 V 10600  t v W15 [E0 10 38 F5] W15 B W15 T
+# S21.5E  11700 H  9750  t V W15 [E0 10 38 F6] W15 B W15 t
+# S21.5E  99999 H 10600  t V W15 [E0 10 38 F7] W15 B W15 T
 
 # Optimized for mini DiSEqC (aka toneburst):
 #
